makefiles/info.inc.mk

```diff
@@ -132,3 +132,6 @@ info-debug-variable-%:
 
 info-toolchains-supported:
 	@echo $(filter-out $(TOOLCHAINS_BLACKLIST),$(TOOLCHAINS_SUPPORTED))
+
+check-toolchain-supported:
+	@exit $(if $(filter $(TOOLCHAIN), $(filter-out $(TOOLCHAINS_BLACKLIST),$(TOOLCHAINS_SUPPORTED))),1,0)
```

Comment:
This is not failing:

```
make -C examples/hello-world/ check-toolchain-supported TOOLCHAIN=llvm QUIET=0 BOARD=hifive1
make: Entering directory '/home/harter/work/git/RIOT/examples/hello-world'
make: Leaving directory '/home/harter/work/git/RIOT/examples/hello-world'
```
I reversed the `1` and the `0` while writing the code… my bad

It should also be added to the `.PHONY` targets.